source/core/src/main/com/csse3200/game/entities/factories/StationFactory.java

```diff
@@ -226,10 +225,15 @@ public static Entity createSubmissionWindow() {
             .addComponent(new StationServingComponent());
     submission.getComponent(InteractionComponent.class).setAsBox(submission.getScale());
     submission.getComponent(PhysicsComponent.class).setBodyType(BodyType.StaticBody);
-    submission.getComponent(TextureRenderComponent.class).scaleEntity();
-    submission.scaleHeight(2f);
     PhysicsUtils.setScaledCollider(submission, 0.3f, 0.2f);
```

Comment:
Same thing on just fixing hitboxes for reliable interaction to help with other teams developement